ReactAndroid/src/main/java/com/facebook/react/ReactActivity.java

```diff
@@ -223,4 +223,13 @@ public void onBackPressed() {
   public void invokeDefaultOnBackPressed() {
     super.onBackPressed();
   }
+
+  @Override
+  public void onNewIntent(Intent intent) {
```

Comment:
@andreicoman11 Any suggestions on how to handle this without needing to add the event to `ReactInstanceManager`?